src/System.Xml.XmlSerializer/src/System/Xml/Serialization/CodeIdentifiers.cs

```diff
@@ -20,20 +20,19 @@ namespace System.Xml.Serialization
     /// </devdoc>
     internal class CodeIdentifiers
     {
-        private Hashtable _identifiers;
-        private Hashtable _reservedIdentifiers;
+        private Dictionary<string, object> _identifiers;
+        private Dictionary<string, object> _reservedIdentifiers;
         private ArrayList _list;
```

Comment:
> But it makes sense to update ArrayList to List<> as well

Great! We just went through and removed all dependencies on `System.Collections.NonGeneric` from all other projects in corefx except `System.Collections.Specialized`. Would be good not to regress, if possible.